huxley/www/js/components/RegistrationView.js

```diff
@@ -5,6 +5,9 @@
 
 'use strict';
 
+// Only necessary until ES2017 becomes standard
```

Comment:
This addresses issue #556. It solves the issue for Safari I know, but I don't have access to IE to test it there (there's no reason it shouldn't work there, I'm just saying I couldn't test it).